src/db/functions/fill_polygon_h3_3.sql

```diff
@@ -0,0 +1,31 @@
+/*This function returns the h3 indexes that are intersecting the borderpoints of a specified geometry*/
+DROP FUNCTION IF EXISTS temporal.fill_polygon_h3_3;
```

Comment:
I think we can add these functions in GOAT core as there I might need them as well. They will be in the same DB so they can be accessed from GOAT-Routing. We should add them to the basic-schema instead of temporal as well.